### PR TITLE
Added a new classification example

### PR DESCRIPTION
**Pull request by tareknaser360**
Applying several classifying models to iris data.
Removed the HTML file and fetched the dataset from mlpack.

### CODE REVIEW

**Review by zoq — COMMENTED** [review]
Can you rename the folder from `iris-classification`-py` to `iris-classification`, just like to make sure we follow the existing structure.

**Comment by tareknaser360** [comment]
> Can you rename the folder from `iris-classification`-py`to`iris-classification`, just like to make sure we follow the existing structure.

Done

File: iris-classification/iris-classification-py.ipynb [hunk 1]
```diff
@@ -0,0 +1,1436 @@
...
+   "cell_type": "markdown",
+   "id": "automatic-culture",
+   "metadata": {},
+   "source": [
+    "## Summary\n",
+    "From the previous quick overview of the data we can conclude basic bullet points:\n",
+    "- The dataset is mostly clean.\n",
+    "- The 'Id' column is not needed.\n",
+    "- 'Species' is our only categorical column.\n",
+    "- There is no missing values or duplicates."
```
Comment (zoq) [inline iris-classification/iris-classification-py.ipynb:521]: This isn't what `df.duplicated().sum()` returns. And 101 and 142 are the same, not sure what the other one is.

File: iris-classification/iris-classification-py.ipynb [hunk 2]
```diff
@@ -0,0 +1,1436 @@
...
...5d0Yrly05mpY+FL0Bn3qdTD9bti6YMfBwrDotWgxraIp0fPPuB/NmQfBRsT/rokuA2jcjnjvOhh/cTSNe+X7mD+9EZuQTyT7Gs1VEB2H21bJVIz6zYgFz7VMKSE2z8b87/NxarWt+2oGB+cenPC4P80/AseCHRkXWghl/hNYl7yISWkNRgJqGtq4S2N33P4DlMRO64ahR28+6jfDZ3fAJ7fCh7+FRa9ijD4fseaz6MMlQ0csfxdjxXvR6tJtmaxonpLWQxoGev74xO/JhCuilTylXmN32YYWk8Ave4IlqddzBTdjevXM1iJYwSbUz+9ANJXDF3+KWc7Xf8EwOxD/vRLe/3X0d+eDXyPeOA/yEwz18dVgJPpN8VYzKnNUwvacO+hcbMgxwZLUVlKC4AceeIAHHngAt9vNWWedxT333NOy7IEHHkjGKXokTTM4pmAaxxQd27LMbrJz/6F/IUvN6/BxfMJD5PRnIbUYckbC1u8S32mt+Tx6U+/KiaYoz4+mO+vjLiZYEDsWJBhR0CdejVHcJgiwejDOepYmaz/CQ05Hn3Jt9Dj542HYKbHpoQCL34Ajb4NDf4t+6uMEPANRv49P/8YwogF0zggAxIYZWANlHX79Uu/QrGYROefVmEDYGHUe4sfX4jeu34ypeXPMolxLEbdNug2zEi2WJhCcM/BMJtdsB19NzLbK/CexR6pb/q3pBuFhZ2MMPrF1I9WMnj4Q/bj74IgdFTaPuBUDBU79R2tVT4B+hyEyB0V7j9sQ66ZjDDi6dYEtBe2cV2m2xF7fXlsJ2mlPtFY/JzrFWGjACeiyWnCvsrtP06IqBMOyJ1iSejMhQK1YEs302XXdnH/C8FPgoGujvznH/hGyhyPmPgYDj4nfftNsjJP/FvPbYQw9GVKLEv6m5DtKuGfqvVhVa8vyE/r9hENzj5C/NZK0i6SkQ+8c91taWkppaeketu5bnHoqt4y9nStGXIk37CXTlkmqyMDQO/78wTCg3tYf588+xBxpQF3wdMJJOIxIAG3afWgmJ0KPIM54Dmwp+G35Mb3AO9WqhXhO+Rdm7xaUsBfNlU+TtYSwBioWLCn58PUD0amTsodHv7Bn/KV1frtIEKpWwIp3MU58BEOo0aeaiYR90aqGO7U3kbzUaxmGQb1zOM6LPsPk3Q6KCWF1YWpnyIDYJX3arFs5NuckJp4wmdpg
```
Comment (zoq) [inline iris-classification/iris-classification-py.ipynb:839]: Is it intentional that the train and test set are the same?

**Review by zoq — COMMENTED** [review]
I really like the notebook, the data insight part is really helpful and interesting.

File: iris-classification/iris-classification-py.ipynb (hunk 2 above)
Comment (tareknaser360) [reply iris-classification/iris-classification-py.ipynb:839]: I don't get the question. 
This is a pair plot. How are the test and train sets the same?

Comment (tareknaser360) [reply iris-classification/iris-classification-py.ipynb:839]: I used TrainTestSplit function to split the dataset
The original dataset is 150 entries
I chose test ratio = 0,2
After splitting, the train set is 120 entries and the test set is 30 entries

Comment (zoq) [reply iris-classification/iris-classification-py.ipynb]: Never mind, I misinterpreted the split.

**Review by zoq — APPROVED** [review]
Looks great to me, thanks for putting this together.

**Comment by zoq** [comment]
Thanks again for the great notebook.

**Comment by tareknaser360** [comment]
> Thanks again for the great notebook.

It was my pleasure! I enjoyed the task. I look forward to contributing more.

**Pull request merged**